src/main/java/uk/gov/hmcts/divorce/divorcecase/model/State.java

```diff
@@ -374,6 +380,13 @@ public enum State {
         Rejected
     ));
 
+    public static final EnumSet<State> PRE_SUBMISSION_STATES = EnumSet.of(
```

Comment:
Aren't awaitingpayment and a few others pre submission states as well?

Reply:
Confirmed that a few states were missing so i've added those in. Thanks Tom